hbase/src/main/java/org/apache/metamodel/hbase/HBaseColumn.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.metamodel.hbase;
+
+import org.apache.metamodel.schema.ColumnType;
+import org.apache.metamodel.schema.ColumnTypeImpl;
+import org.apache.metamodel.schema.MutableColumn;
+import org.apache.metamodel.schema.SuperColumnType;
+import org.apache.metamodel.schema.Table;
+
+public final class HBaseColumn extends MutableColumn {
+    public static final ColumnType DEFAULT_COLUMN_TYPE_FOR_ID_COLUMN = new ColumnTypeImpl("BYTE[]",
+            SuperColumnType.LITERAL_TYPE);
+    public static final ColumnType DEFAULT_COLUMN_TYPE_FOR_COLUMN_FAMILIES = ColumnType.LIST;
+
+    private final String columnFamily;
+    private final String qualifier;
+
+    public HBaseColumn(final String columnFamily, final Table table) {
+        this(columnFamily, null, table, -1);
+    }
+
+    public HBaseColumn(final String columnFamily, final String qualifier, final Table table) {
+        this(columnFamily, qualifier, table, -1);
+    }
+
+    public HBaseColumn(final String columnFamily, final Table table, final int columnNumber) {
+        this(columnFamily, null, table, columnNumber);
+    }
+
+    public HBaseColumn(final String columnFamily, final String qualifier, final Table table, final int columnNumber) {
```

Comment:
This is a lot of constructors :-) I would only add the ones you use yourself, and then make the class non-public and final.

Reply:
I removed the unused constructors. For now I left the class public however, because it represents the manner in which an HBase column works. I think it is possible to use a Column, but then you need some logic to deduce the actual HBase column from the normal Column, using a colon to split the Column's name on and get an HBaseColumn based on that.